metabase/metabase
# Audit v2 Collections Frontend

Closes https://github.com/metabase/metabase/issues/30988

### Description

Adds a special Icon to audit collection headers and removes the "make collection official" option for them.

![Screen Shot 2023-05-24 at 1 31 54 PM](https://github.com/metabase/metabase/assets/30528226/ecce2780-9e97-43ee-852c-2db69ae19532)

### How to verify

- Create an "Instance Analytics" collection (the entity mock will make it a "special" collection)
- See that it gets a header icon (placeholder beaker for now)
- see that the icon has a tooltip
- see that on EE you cannot make the collection official.

### Checklist
